.gitignore

```diff
@@ -1,5 +1,6 @@
 # The directory Mix will write compiled artifacts to.
 /_build/
+.elixir_ls/
```

Comment:
When running dyalizer in my machine this appeared. I assume it should be ignored